### PR TITLE
[StatusChatInput]: fixed focus and reply component lost 

### LINKED ISSUE #9139: Pressing the Up arrow when editing a message loses the cursor
# Bug Report

## Description

If you're in edit mode of a message and press the up arrow, the field loses focus and you need to re-click on it.

The same goes for going in edit mode using the up arrow when you're in the normal chat input. So doing up arrow twice loses the focus.

### Additional Information

- Status desktop version: Master (`93c90b84362c6a1baaeb6d24cdc25d200a148bae`)

### LINKED ISSUE #9419: Chat: reply message is gone when replying with an image
### Description

1. go to community
2. join any channel
3. click to any message to reply
4. paste an image to have it as a reply

As result, message that u were about to reply, is gone from the input box

https://user-images.githubusercontent.com/82375995/216570930-33a9c17a-a507-4b91-b986-345961a7c676.mov

### PR DESCRIPTION
**Pull request by alexandraB99**
- Fixed reply message is gone when adding image
- Fixed pressing up arrow key when editing loses the cursor

Closes #9419
Closes #9139

### What does the PR do
StatusChatInput various fixes

### Affected areas
StatusChatInput

### CODE REVIEW

**Review by caybro — APPROVED** [review]
LGTM

**Review by jrainville — APPROVED** [review]

File: ui/imports/shared/status/StatusChatInput.qml [hunk 1]
```diff
@@ -1245,6 +1240,15 @@ Rectangle {
                             bottomPadding: 9
                             leftPadding: 0
                             padding: 0
+                            Keys.onUpPressed: {
+                                if (isEdit) {
```
Comment (alexjba) [inline ui/imports/shared/status/StatusChatInput.qml:1244]: This breaks the emoji/mention suggestions key navigation 😟

**Review by micieslak — COMMENTED** [review]
Commit message could be a bit more descriptive ;) ;)

File: ui/imports/shared/status/StatusChatInput.qml (hunk 1 above)
Comment (alexandraB99) [reply ui/imports/shared/status/StatusChatInput.qml:1244]: @alexjba good catch! fixed

**Comment by alexandraB99** [comment]
> Commit message could be a bit more descriptive ;) ;)

updated

**Review by micieslak — APPROVED** [review]
:+1:

**Review by alexjba — APPROVED** [review]
LGTM! 👍

**Pull request merged**